src/bundle/Resources/views/themes/admin/ui/form_fields.html.twig

```diff
@@ -247,10 +255,77 @@
 
 {% block form_errors -%}
     {%- if errors|length > 0 -%}
+        {%- set form_errors_attr = form_errors_attr|default({}) -%}
+        {%- set form_errors_attr = form_errors_attr|merge({class: (form_errors_attr.class|default('') ~ ' ibexa-form-error')|trim}) -%}
```

Comment:
```suggestion
        {%- set form_errors_attr = form_errors_attr|default({})|merge({class: (form_errors_attr.class|default('') ~ ' ibexa-form-error')|trim}) -%}
```
maybe?

Reply:
From what I have checked, in this case we would need default in 2 places:
```suggestion
        {%- set form_errors_attr = form_errors_attr|default('')|merge({class: (form_errors_attr|default('').class|default('') ~ ' ibexa-form-error')|trim}) -%}
```
This is why I added:
`{%- set form_errors_attr = form_errors_attr|default({}) -%}` above instead.

Reply:
Discussed in private - `|default()` in twig works like "optional chaining". 
https://twigfiddle.com/amxe65